### ISSUE TITLE
Form element do not have appropriate labels

### ISSUE THREAD

**Issue by eadehemingway** [fac-13/Team-GIPHT-Todo#36]
the accessibility audit says that the form elements should have labels, we had this same issue which is tricky because the form elements that are created in the dom arent in the html. 
to get around this you have create label tags in the dom and append the form inputs into the label tags before appending the whole thing into the html...

**Comment by tspeed90** [comment]
Thanks, added it! I had done for the mark and delete buttons, but forgot for the form. @eadehemingway
